aiidalab/app.py

```diff
@@ -752,12 +839,18 @@ def _refresh_versions(self):
                 if self.include_prereleases or not parse(version).is_prerelease
             ]
 
-    @throttled(calls_per_second=1)
+    def _refresh_dependencies_to_install(self):
+        self.dependencies_to_install = self._app.find_dependencies_to_install(
+            self.version_to_install
+        )
+
+    @throttled(interval=1)
```

Comment:
I think this can be 10, means the app manager update every 10s.

Reply:
This is an old issue if you checkout to the main branch. You can find the app that is already installed. It will keep on flashing in the 1s interval, which is very annoying.

I can open a separate PR for this. What you think? @danielhollas @yakutovicha

Reply:
Yes, let's not complicate this PR more. Please revert these changes and create an issue.